CODEOWNERS

```diff
@@ -1 +1,4 @@
 /features.yaml @linsun @dcberg @louiscryan @smawson @duderino @nrjpoddar
+. @linsun @dcberg @louiscryan @smawson @duderino @nrjpoddar
+/features @istio/maintainers
```

Comment:
I think this will also request review from everyone

Reply:
I think that's ok. Theoretically this repo is fairly low traffic. I think TOC should be the approvers for the top level. For the features directory, I think it's ok for any maintainer in any work group to be able to approve a change. Unfortunately, I don't think GitHub gives us the flexibility to give someone approval rights without also tagging them as a reviewer.